front/app/components/CustomFieldsForm/Fields/SingleSelectField.tsx

```diff
@@ -42,15 +55,32 @@ const SingleSelectField = ({
           scrollErrorIntoView={scrollErrorIntoView}
         />
       ) : (
-        <RadioGroup name={question.key}>
+        <RadioGroup name={question.key} padding="0px">
+          <ScreenReaderOnly>
+            <legend>{localize(question.title_multiloc)}</legend>
```

Comment:
Didn't realize we needed this for accessibility :+1: I'll remember for next time :)